### DISCUSSION TITLE
Feedback/Not an issue: DX11 (Merged) decreased loading times around 75% in my case. What about you?

### DISCUSSION THREAD

**Discussion by notNSANE** [bcssov/IronyModManager#139] [Show and tell]
Again, not an issue, just curious about other people's numbers. These are my numbers in minutes:

DX9: 5+ minutes
DX9 (Merged): around 2 minutes
DX11: 2.5 minutes
DX11 (Merged): 0.5 minutes (!!)

So happy about this! Using a Ryzen 1600.

**Comment by bcssov** [comment]
Going to put a label on it so that stale bot doesn't close this issue.

EDIT: You might want to also inquire on the [Stellaris Modding Den](https://discord.gg/bHVez2C), Irony has a channel there where I and many of my users frequent.

**Comment by bcssov** [comment]
DX11 Basic Merge Load 1 38:73s
DX11 Basic Merge Load 2 37:77s
DX11 Advanced Merge Load 1 36:97s
DX 11Advanced Merge Load 2 35:89s

Using i7-8700k >270ish mods.
